operator/pkg/central/extensions/reconcile_tls.go

```diff
@@ -69,7 +69,12 @@ func (r *createCentralTLSExtensionRun) Execute(ctx context.Context) error {
 	if err := r.ReconcileSecret(ctx, "scanner-db-tls", scannerEnabled && !shouldDelete, r.validateScannerDBTLSData, r.generateScannerDBTLSData, true); err != nil {
 		return errors.Wrap(err, "reconciling scanner-db secret")
 	}
+	return nil // ReconcileInitBundleSecrets not called due to ROX-9023. TODO(ROX-9969): call after the init-bundle cert rotation stabilization.
+}
 
+// ReconcileInitBundleSecrets is only exported temporarily to silence the static checker while it's unused.
+// TODO(ROX-9969): make private after the init-bundle cert rotation stabilization.
+func (r createCentralTLSExtensionRun) ReconcileInitBundleSecrets(ctx context.Context, shouldDelete bool) error {
```

Comment:
nit: `//nolint` to stop the linter from reporting the unused function? 🤔 
See: https://golangci-lint.run/usage/false-positives/

Reply:
Added `lint:ignore` (suggested `nolint` for some reason doesn't work with `staticcheck`)